styles/styles.css

```diff
@@ -0,0 +1,91 @@
+/* avatar */
+
+.avatar {
+  border-radius: 50%;
+  margin: 0.5rem;
+  height: 50px;
+  width: 50px;
+}
+
+.avatar-sm {
+  height: 40px;
+  width: 40px;
+}
+
+.avatar-md {
+  height: 80px;
+  width: 80px;
+}
+
+.avatar-lg {
+  height: 120px;
+  width: 120px;
+}
+
```

Comment:
You can give hights and width with rems. Just saying ,as it might be helpful in future.

Reply:
Giving images the sizes in pixels I thought would be better. I look into it again once I'll be building image component.